tox.ini

```diff
@@ -1,26 +1,11 @@
 [tox]
-envlist = py35,py36,py37,py38,lint,check-formatting,manifest
+envlist = py35,py36,py37,py38,manifest
 isolated_build = True
 
 [testenv]
 extras = tests
 commands = python -m pytest {posargs}
 
-[testenv:lint]
-basepython = python3.8
-deps =
-    pre-commit
-commands = pre-commit run --all-files
-
-; This env just runs `black` and fails tox if it's not formatted correctly.
-; If this env fails on CI, run `tox -e format` locally in order to apply changes.
-[testenv:check-formatting]
```

Comment:
Actually - I'm a bit brain dead rn so I'm not fully understanding why - but #54 had succeeded at the pre-commit step but [failed](https://github.com/econchick/interrogate/pull/54/checks?check_run_id=1207750022) at the `tox -e check-formatting` step.

Reply:
ah wait I know why - because these versions weren't updated 🤦